### ISSUE TITLE
Phrasal verbs: drop directional particles? 

### ISSUE THREAD

**Issue by cbonial** [amrisi/amr-guidelines#11]
We have decided to use a simple, lexical verb entry where this only differs from a phrasal verb in aspectual qualities (e.g. eat vs. eat up).  In some cases, e.g. lay down, come back, we can use the simple lexical verb, but should the particle be included at all?  e.g. ":direction back/down"?  Or should this be simplified into using just the lexical verb? I was including the directional particle in cases where roleset included :argm DIR or a numbered arg for direction (e.g. "come"), and leaving it out where the roleset didn't include this, (e.g. "sit/lay down").

**Comment by nschneid** [comment]
One of my students raised this issue with "drive in" (e.g., "My parents drove in for Thanksgiving"). I vote for `:direction (i2 / in)` or similar.

**Comment by kevincrawfordknight** [comment]
Agreed that this is not covered by drive-XX alone.  ":direction (i2 / in)" seems okay to me, though this gives reasoning engines yet another thing they aren't expecting :)  The sentence really seems to mean "arrived by driving", so another idea would be to use those predicates.
